scripts/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+  // This is an alias to @tsconfig/node16: https://github.com/tsconfig/bases
+  "extends": "ts-node/node16/tsconfig.json"
```

Comment:
Interesting didn't know about this.

Reply:
Me neither. I didn't really want to have two separate configs, but it was taking too long to troubleshoot the other one

Reply:
At least with this extends feature we're not actually _maintaining_ a separate set of configs